spec/features/12_admin_deletes_any_shoe_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+  # [] As an admin
+  # [] I want to delete the information of any shoe
+
+feature "admin can delete shoe information" do
+  scenario "admin deletes shoe successfully, for which they are also the owner" do
+    user = FactoryGirl.create(:user)
+    shoe = FactoryGirl.create(:shoe, user: user)
+    user.admin = true
```

Comment:
You may want to update your `user` factory to create an `:admin`. No need to update for now, but check out this article on more advanced FactoryGirl usage: http://arjanvandergaag.nl/blog/factory_girl_tips.html